packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -73,23 +78,38 @@ export function useMenuTrigger(props: MenuTriggerAriaProps, state: MenuTriggerSt
     }
   };
 
+  let {longPressProps} = useLongPress({
+    isDisabled: false,
```

Comment:
Shouldn't this be set to just `isDisabled` instead of `isDisabled: false`? Wouldn't this mean the long press events would fire even if `isDisabled` is true?

Reply:
Makes sense, I'll fix this in my incoming commit. Thank you!

Reply:
@LFDanLu fixed in the newest commit